examples/values_stateful.yaml

```
@@ -0,0 +1,37 @@
+# stateful deployment for multiple replicas with persistent state per replica
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)